staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -5001,15 +5005,6 @@ func translateTimestampSince(timestamp metav1.Time) string {
 	return duration.HumanDuration(time.Since(timestamp.Time))
 }
 
-// formatEventSource formats EventSource as a comma separated string excluding Host when empty
-func formatEventSource(es corev1.EventSource) string {
-	EventSourceString := []string{es.Component}
-	if len(es.Host) > 0 {
```

Comment:
Omit `EventSource.Host` in such way, should we care backward compatibility?

Reply:
Host is deprecated in 1.18 or less, can't be set in 1.19. How far back in the past kubectl compatibility goes? 2 or 3 versions back?

Reply:
Given this is targeting 1.20, and you can't set that in 1.19 this ok change.

Reply:
We support only +/- 1 version